packages/bahmni-widgets/src/commandPalette/styles/CommandPalette.module.scss

```
@@ -0,0 +1,423 @@
+}
+
+.annotationSuggestionPrefix {
+  display: inline-flex;
+  align-items: center;
+  padding: 1px $spacing-02;
+  background: $layer-03;
+  border: 1px solid $border-subtle-02;
+  border-radius: 4px;
+  color: $interactive;
+  @include type-style('code-01');
+  font-size: 0.75rem;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)